source/dubregistry/web.d

```diff
@@ -363,8 +386,28 @@ class DubRegistryWebFrontend {
 
 	private void updatePackageList()
 	{
-		setTimer(30.seconds, &updatePackageList);
```

Comment:
I assume extending the update period from 30 seconds to the 30 minute watcher in app.d was done on purpose.

Reply:
Do you mean that something was changed in this PR in that regard? It was 30 seconds and it still is (line 70). I just changed this to a periodic timer stead of starting a new one every time.

Reply:
Ah didn't see that. I was referring to the global timer.